.circleci/config.yml

```diff
@@ -13,12 +13,6 @@ jobs:
       - image: quay.io/pypa/manylinux1_i686
     steps:
       - checkout
-      - run:
-          name: Install dependencies for Python 3.5
```

Comment:
I feel like dropping Python 3.5 is beyond the scope of this PR.

Reply:
I would also suggest to wait with dropping it for at least until 3.2 is out, but better if not even ~1-2 months after the release to make it easier to do bugfix releases.